client/src/js/SM/Library.js

```diff
@@ -47,7 +47,11 @@ SM.Library.ChecklistGrid = Ext.extend(Ext.grid.GridPanel, {
         mapping: 'detail.vulnDiscussion'
       },
       {
-        name: 'ccis'
+        name: 'ccis',
+        // Convert the CCIs array into a joined string when loading into the store
+        convert: function(v) {
```

Comment:
Could be simplified to:
```javascript
convert: v => v.map(i => i.cci).join('\n')
```
`v` is always present and an array, so there's no need for a conditional chain.

Reply:
done!